src/openrct2/rct2/RCT2.h

```diff
@@ -972,7 +972,7 @@ namespace RCT2
         uint8_t RCT1WaterColour;
         uint8_t Pad01358842[2];
         RCT12ResearchItem ResearchItems[Limits::MaxResearchItems];
-        uint16_t MapBaseZ;
+        uint16_t kMapBaseZ;
```

Comment:
Shouldn't this have been removed instead?

Reply:
As understand it, it's part of the original save format, right?

Reply:
You mean it needs to be kept so it reads things in order? If that is the case, then we should rename or comment that this was made unused

Reply:
We didn’t do that for all the other cases, though. The variable should be reverted, but a comment is not necessary.